src/commands/react-native/README.md

```diff
@@ -100,6 +100,37 @@ This allows Datadog to create proper URLs such as:
+| `--bundle`          | Required  | Should be set as the path to your generated JS bundle file, `main.jsbundle` for iOS and `index.android.bundle` for Android.                                                                                                                                                                                                                                                 |
+| `--sourcemap`       | Required  | Should be set to the path to your generated source map file, `main.jsbundle.map` for iOS and `index.android.bundle.map` for Android.                                                                                                                                                                                                                                        |
+| `--app`             | Required  | The name of the target app in AppCenter. Must match the format OrganizationName/AppName.                                           |
+| `--deployment`      | Required  | The name of the deployment in AppCenter.             |
+
+The following optional parameters are available:
+
+| Parameter          | Default | Description                                                                                                                                                                                                                     |
+|--------------------|---------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `--disable-git`    | False   | Prevents the command from invoking git in the current working directory and sending repository-related data to Datadog (such as the hash, remote URL, and paths within the repository of sources referenced in the source map). |
+| `--dry-run`        | False   | It runs the command without the final step of uploading. All other checks are performed.                                                                                                                                        |
+| `--repository-url` | Empty   | Overrides the remote repository with a custom URL. For example, `https://github.com/my-company/my-project`.                                                                                                                     |
+| `--build-version`  | 1      | Used to avoid overwriting your source maps by accident. Only one upload is needed for a specific `build-version` and `service` combination. Subsequent uploads are ignored until the `build-version` changes. This should not be necessary for codepush unless you uploaded the wrong source maps by mistake.   | 
```

Comment:
```suggestion
| `--build-version`  | 1      | Used to avoid overwriting your source maps by accident. Only one upload is needed for a specific `build-version` and `service` combination. Subsequent uploads are ignored until the `build-version` changes. This should not be necessary for CodePush unless you uploaded the wrong source maps by mistake.   | 
```

Reply:
I can't tell if this is talking about the general concept of pushing code, or a specific service called CodePush. If it is the service, please capitalize it as CodePush. If it is the concept of pushing code, please use "code push".